service/matching/matchingEngine.go

```diff
@@ -806,7 +806,7 @@ func (e *matchingEngineImpl) createPollForActivityTaskResponse(
 	}
 
 	response.TaskToken, _ = e.tokenSerializer.Serialize(token)
-	response.Attempt = common.Int32Ptr(int32(token.ScheduleAttempt))
+	response.Attempt = int32(token.ScheduleAttempt)
```

Comment:
I'm guessing it's an int64?
I don't believe this came up in the string/int cast linting I saw, but mind triple checking that this cast is sane? (I wish go casts weren't so yolo)

Reply:
Yeah, those are not consistent across codebase and there multiple conversions back and forth like this.
I have converted all attempt fields to `int32` in newly introduced proto IDLs. 
I think we should make them consistent internally as well. But this is another task.